document-service-api/src/api/openapi.yaml

```diff
@@ -0,0 +1,179 @@
+---
+openapi: 3.0.0
+info:
+  title: Appeals Document Service API Documentation
+  version: 1.0.0
+  license:
+    name: www.foundry4.com
+servers:
+  - url: http://localhost:3001/
+paths:
+  "/api/v1/{applicationId}":
+    description: Resources related to application specific documents
+    get:
+      tags:
+        - Document
+      operationId: getApplicationDocuments
+      description: Obtains all document data related to a given application ID
+      parameters:
+        - "$ref": "#/components/parameters/applicationId"
+      responses:
+        '200':
+          description: Successfully retrieved documents for the given Id
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/DocumentList"
+        '400':
+          description: Unable to process this request.
+        '404':
+          "$ref": "#/components/responses/NotFound"
+    post:
+      tags:
+        - Document
+      operationId: postDocument
+      description: Creates a new file upload against the specified application Id
+      parameters:
+        - "$ref": "#/components/parameters/applicationId"
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                file:
+                  type: string
+                  format: binary
+      responses:
+        '202':
```

Comment:
I think we need a content: attribute for the responses that do give json responses e.g.
<img width="2258" alt="Screenshot 2020-12-14 at 12 20 05" src="https://user-images.githubusercontent.com/56199648/102080951-48ae2e80-3e07-11eb-9527-25edf69d0485.png">

Reply:
Ok - speak to Lee as author of document